gledki.go

```diff
@@ -330,15 +330,15 @@ func (t *Gledki) findRoots(roots []string) error {
 				t.Roots = append(t.Roots, byCwd)
 				continue
 			} else {
-				return fmt.Errorf("gledki root directory '%s' does not exist! You have to create it. ", byCwd)
+				return fmt.Errorf("Gledki root directory '%s': %w! You have to create it.", byCwd, os.ErrNotExist)
 			}
 		}
 
 		if dirExists(root) {
 			t.Roots = append(t.Roots, root)
 			continue
 		} else {
-			return fmt.Errorf("Gledki root directory '%s' does not exist!", root)
+			return fmt.Errorf("Gledki root directory '%s': %w!", root, os.ErrNotExist)
```

Comment:
```suggestion
			return fmt.Errorf("Gledki root directory '%s': %w", root, os.ErrNotExist)
```
Using an `!` is like shouting to the user. I feel that's impolite :)